app/src/renderer/views/_mixins.pug

```diff
@@ -0,0 +1,90 @@
+//-
+  Kap Pug Mixins ⚡️
+  ----
+  - Re-usable components in the kap markup
+
+
+//- **
+    name: icon
+    description: An easier method for fetching SVGs in markup.
+    @param {string} name - Name of the svg file you're calling
+    example: +icon('crop')
+mixin icon(name)
+  img(src='/app/static/'+name+'.svg')&attributes(attributes)
```

Comment:
Not sure whether you can use template literals in Pug templates, but would look nicer if possible:

```js
img(src=`/app/static/${name}.svg`)&attributes(attributes)
```